src/components/KeyboardShortcuts.vue

```diff
@@ -0,0 +1,39 @@
+<template>
+  <closeable-dialog v-model="keyboardStore.settingsOpen">
+    <v-card>
+      <v-card-title class="d-flex flex-row align-center"
+        >Keyboard Shortcuts</v-card-title
```

Comment:
Another prettier styling snafu here. Not sure if this is something we can configure.

Reply:
Fixed.  🤷 I'm getting used to it.

Reply:
It might be fixed in a newer version of prettier, or there's some vue-prettier plugin we need.